<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-T81UTAF" name="testNGpack.Flag4" tests="3" failures="0" timestamp="2022-08-24T20:48:12 IST" time="0.024" errors="1">
  <testcase name="actTimeLogin" time="0.024" classname="testNGpack.Flag4">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at testNGpack.Flag4.actTimeLogin(Flag4.java:10)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- actTimeLogin -->
  <system-out/>
  <testcase name="actitimeCreateUser" time="0.000" classname="testNGpack.Flag4">
    <skipped/>
  </testcase> <!-- actitimeCreateUser -->
  <system-out/>
  <testcase name="actitimeLogout" time="0.000" classname="testNGpack.Flag4">
    <skipped/>
  </testcase> <!-- actitimeLogout -->
  <system-out/>
</testsuite> <!-- testNGpack.Flag4 -->
